test/test_prelude.py

```diff
@@ -85,10 +91,16 @@ def test_non_default_mask(self):
         mask = np.ones(self.phase_e1.shape)
 
         # Call prelude with mask (is_unwrapping_in_2d is also used because it is significantly faster)
-        unwrapped_phase_e1 = prelude(self.phase_e1, self.mag_e1, self.affine_phase_e1, mask, is_unwrapping_in_2d=True)
+        unwrapped_phase_e1 = prelude(
```

Comment:
@jidicula do you have an automatic formatting in your IDE? if so maybe we should discuss formatting conventions. I personally don't mind how the code was written before-- it was still compliant with PEP8. PEP8 is fairly flexible when it comes to how code should be wrapped, so it's a matter of internal convention (i tend to like concise code, not spread out across multiple lines)

Reply:
Yes, sorry about that! I'll disable it in my local settings for this repo.

Reply:
Shall I try to roll back these formatting changes in a fixup commit?

Reply:
if you don't mind-- otherwise no big deal- it's still clean code 😊